### ISSUE TITLE
Null street at checkout

### ISSUE THREAD

**Issue by okolya** [netresearch/dhl-module-ui#5]
https://github.com/netresearch/dhl-module-ui/blob/8454c34b969c1880790db8ee0dc71857c82d7358/view/frontend/web/js/view/input/shopfinder.js#L77

When new session without addess is activated at checkout page then this line produces error.

And shopfinder button is not available.
![Screenshot_at_15-10-35](https://user-images.githubusercontent.com/2494612/106480157-5e58df80-64ab-11eb-8bb7-a601dddea5ed.png)

**Comment by okolya** [comment]
Solution: replace with line
```
this.searchStreet(address.street ? address.street.join(' ') : '');
```

**Comment by mam08ixo** [comment]
Thank you!
The issue has a fix that is waiting for a release. It did not make it into the upcoming 1.5.0 though.
I will keep this issue open until the fix was released.

**Comment by okolya** [comment]
Do you mean the release of **dhl/module-ui** or **dhl/shipping-m2** ?
I am wondering because the **dhl/shipping-m2** moduel has already 1.5.0 version.
And the **dhl/module-ui** module  has only 1.2.2 .

**Comment by mam08ixo** [comment]
Sorry for the confusion. I was referring to the metapackage `dhl/shipping-m2` which is indeed already available in version 1.5.0 on GitHub but not yet on Magento Marketplace.

**Comment by okolya** [comment]
Am I right, that you are going to release dhl/module-ui 1.2.3 and update dhl/shipping-m2 ?
The reported issue has occured by dhl/shipping-m2 wich was installed from github (1.5.0).

![Screenshot_at_17-19-18](https://user-images.githubusercontent.com/2494612/106486205-b1ce2c00-64b1-11eb-8020-85c344466acc.png)

I have found how to apply fix inside my template. But it overwrite complete shopfinder.js component. 
Unfortunately there is no way how to to do it with mixin logic.

Are you planning to apply fix to resolve this issue?

**Comment by mam08ixo** [comment]
The fix will be included with the next `dhl/module-ui` version as part of the next metapackage release.

**Comment by mam08ixo** [comment]
Fixed in version 1.2.3
